main.go

```diff
@@ -12,13 +13,17 @@ import (
 	"log"
 	"os"
 
+	"github.com/google/go-containerregistry/pkg/name"
```

Comment:
nit: pick a better identifier for this package as "name" is pretty commonly used as a variable etc. Maybe `crname`?